### ISSUE TITLE
Tickets Do Not Appear in Ticket Assignments Manager with New Event on Fresh Install

### ISSUE THREAD

**Issue by tn3rb** [eventespresso/event-espresso-core#2712]
Here's the data from my `wp_esp_datetime` table

DTT_ID | EVT_ID | name | description | EVT_start | EVT_end | reg_limit | sold | reserved | is_primary | order | parent | deleted
--- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- 
1 | 9 | WSL 1 | yada yada yada | 2020-05-20 12:00:00 | 2020-05-20 21:00:00 | -1 | 0 | 0 | 0 | 1 | 0 | 0

Here's the data from my `wp_esp_ticket` table

TKT_ID | TTM_ID | name | description | qty | sold | reserved | uses | required | min | max | price | start_date | end_date | taxable | order | row | is_default | reverse_calculate | wp_user | parent | deleted
--- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | ---  | --- 
1 | 0 | Free Ticket |  | 100 | 0 | 0 | 	- 1 | 0 | 0 | 	- 1 | 0.000 | 2020 - 04 - 21 00: 12: 07 | 2020 - 05 - 21 03: 59: 00 | 0 | 0 | 1 | 1 | 0 | 1 | 0 | 0

Here's the data from my `wp_esp_datetime_ticket` table

DTK_ID | DTT_ID | TKT_ID
--- | --- | --- 
1 | 1 | 1

And here's a screenie showing the issue in action:

![TAM bug 2020-04-20.gif](https://images.zenhubusercontent.com/55c8c0f14f7a08fbb7f431cd/c8151137-f53d-4190-bdfa-5a18d73bb872)

So basically, regardless of which TAM button the user chooses (date relations, ticket relations, or ALL relations) the view should be the same for an event with only a single date and a single ticket.

**Comment by manzoorwanijk** [comment]
After debugging, it turned out that it's not the tickets that are not loaded, rather it's the dates that are not loaded.
**Reason**: When a new event is created it has the only date set for next month. When TAM loads, we set the value for `datesByMonth` filter/dropdown to the current month which does not have any dates. But then, you can't trigger the month change because there is only one item in the dropdown.

**Solutions**:

1. Set the default selected month to the immediate next datetime
2. Add the current month anyway to the dropdown list.

I am more inclined towards the first solution.

**Comment by nicolad** [comment]
I also agree with the first solution, because if I understand correctly that way we will have displayed any relations.

**Comment by manzoorwanijk** [comment]
> I understand correctly that way we will have displayed any relations.

No, the relations will still be for a single month.
